packages/angular/src/app/services/screen.service.ts

```diff
@@ -1,18 +1,34 @@
 import { Output, Injectable, EventEmitter } from '@angular/core';
-import { BreakpointObserver, Breakpoints } from '@angular/cdk/layout';
+import { BreakpointObserver } from '@angular/cdk/layout';
 
-import { Subscription } from 'rxjs';
+import { ReplaySubject, Subscription} from 'rxjs';
+
+const Breakpoints = {
```

Comment:
Why do you create the breakpoints? What's the difference with default value?

Reply:
These are the requirements of our designer: breakpoints must be same as in the bootstrap framework.

Reply:
it looks wrong because we are using material. They use different values https://material.angular.io/cdk/layout/overview#predefined-breakpoints

Reply:
You can also see that there is no value less than xSmall